README.md

```diff
@@ -12,7 +12,7 @@ Supported Zeebe versions
 > **NOTE**: version 1.0 is incompatible with Zeebe versions pre 0.23.x
 
 - 0.23.x
-- 0.24.x
```

Comment:
It should still support 0.24.x, so no need to remove this line. In fact, it should be compatible with any version > 0.23.x